index.html

```diff
@@ -70,17 +70,76 @@
   homeButton.options.position = 'topright'
   homeButton.addTo(map)
 
-  L.tileLayer('http://{s}.tile.osm.org/{z}/{x}/{y}.png', {
-      attribution: '&copy; <a href="http://osm.org/copyright">OpenStreetMap</a> contributors'
-    }).addTo(map);
+  // Look if world coastline map ist locally available and use it
+  var tilemapUrl = false;
+  var tilemapMeta = false;
+  $.ajax({
+    // path of list of maps in Signal K server
+    url: "/signalk/v1/api/resources/charts",
+    async: false,
+    success: function(data) {
+      // Not the data we expected? URL wrong?
+      if (data === null) {
+        console.log("Signal K server not found.")
+        return;
+      }
+      if (typeof data !== 'object') {
+        console.log("Signal K server not found.")
+        return;
+      }
+
+      // Is the world coastline map installed?
+      if (typeof data['world-coastline.mbtiles'] !== 'undefined') {
+        // this will be "/charts/world-coastline.mbtiles/{z}/{x}/{y}"
+        console.log("Show the world coastline map.")
+        tilemapUrl = data['world-coastline.mbtiles'].tilemapUrl;
+        tilemapMeta = {
+          attribution: '&copy; <a href="http://osm.org/copyright">OpenStreetMap</a> contributors',
+          maxZoom: 11,
+          tileSize: 1024,
+          zoomOffset: -2,
+        };
+        return;
+      }
+      // Is any other map installed on Signal K server?
+      if(!$.isEmptyObject(data)) {
+        // Take one map we randomly find in list of maps
+        $.each(data, function(index, firstMap) {
+          console.log("Map used: ", firstMap.identifier);
+          tilemapUrl = firstMap.tilemapUrl;
+          tilemapMeta = {
+            attribution: '&copy; <a href="http://osm.org/copyright">OpenStreetMap</a> contributors',
```

Comment:
A random chart is probably not OpenStreeMap, empty attribution maybe a better choice? 

Can't remember off hand if SK chart api has some attribution/copyright field, should consider one.